File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/02.Conditional-Statements/02-exercise/07.shopping.js
function shopping(input) {

    const budget = Number(input[0]);
    const videoCardsCount = Number(input[1]);
    const processorsCount = Number(input[2]);
    const ramMemoryCount = Number(input[3]);

    const oneVideoCardPrice = 250;
    const videoCardsPrice = videoCardsCount * oneVideoCardPrice;
    const processorsPrice = processorsCount * (videoCardsPrice * 0.35);
    const ramMemoryPrice = ramMemoryCount * (videoCardsPrice * 0.1);
    let totalPrice = videoCardsPrice + processorsPrice + ramMemoryPrice;

    if (videoCardsCount > processorsCount) {
        totalPrice *= 0.85;
    }

    const difference = Math.abs(budget - totalPrice);

    if (budget >= totalPrice) {
        console.log(`You have ${difference.toFixed(2)} leva left!`);
    } else {
        console.log(`Not enough money! You need ${difference.toFixed(2)} leva more!`);
    }

}
shopping(["900", "2", "1", "3"]);

File: Python_Basics/Homeworks-And-Labs/02.conditional_statements/02.exercise/07.shopping.py
budget = float(input())
videoCardsCount = int(input())
processorsCount = int(input())
ramMemoryCount = int(input())

oneVideoCardPrice = 250
videoCardsPrice = videoCardsCount * oneVideoCardPrice
processorsPrice = processorsCount * (videoCardsPrice * 0.35)
ramMemoryPrice = ramMemoryCount * (videoCardsPrice * 0.1)
totalPrice = videoCardsPrice + processorsPrice + ramMemoryPrice

if videoCardsCount > processorsCount:
        totalPrice *= 0.85

difference = abs(budget - totalPrice)

if budget >= totalPrice:
    print(f'You have {difference:.2f} leva left!')
else:
    print(f'Not enough money! You need {difference:.2f} leva more!')